beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/context/CommonBeans.java

```diff
@@ -17,6 +17,13 @@
 
 import java.util.List;
 
+import com.expediagroup.beekeeper.scheduler.apiary.filter.*;
```

Comment:
Imports need to be fully qualified and in the right order (we need to document this setup for OSS...)

Reply:
I can update the contributing doc for this - is there a linter or editorconfig that we use standard to enforce a specific code style?

Reply:
We didn't have anything to do import ordering but after reviewing this I remembered seeing it in another project of ours so I have raised a PR against the parent pom (which this project uses) to sort that out - https://github.com/ExpediaGroup/eg-oss-parent/pull/8. Once that's merged we can upgrade the parent pom here and then all the import ordering and optimisation should be sorted out. We have a very high level editforconfig file at https://github.com/ExpediaGroup/new-project/blob/master/.editorconfig which we could add to this project if it would be useful although we've found it pretty limited in what it supports.